modules/ROOT/pages/configuring-properties.adoc

```diff
@@ -198,7 +198,7 @@ A common configuration use case is to set the file to depend on a property (for
 <configuration-properties file="${env}-properties.yaml"/>
 ----
 
-This way, the value of the property `env` determines which file to use to load the configuration properties. That `env` property can be set by a global property, system property, or environment property. Global properties are a way to define default values for properties, that can be overriden by any other type of property (e.g. a System Propery).
+This way, the value of the property `env` determines which file to use to load the configuration properties. That `env` property can be set by a global property, system property, or environment property. You can use global properties as a way to define default values for configuration properties. System and environment properties with the same name as a global property will override that Global Property.
```

Comment:
Please make Global Property lowercase in the last sentence.

Reply:
```suggestion
This way, the value of the property `env` determines which file to use to load the configuration properties. That `env` property can be set by a global property, system property, or environment property. You can use global properties as a way to define default values for configuration properties. System and environment properties with the same name as a global property will override that global property.
```